### PR TITLE
Clickable menu

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: app/directives/meetings/conference-header.js [hunk 1]
```diff
@@ -29,7 +29,7 @@
                         return basePath + $location.path();
                     },
 
-                    isSelected : function(name, exact) {
+                    isSelected : function(name, exact, t) {
```
Comment (sbilodeau) [inline app/directives/meetings/conference-header.js]: what is `t`?

Comment (blaisefonseca) [reply app/directives/meetings/conference-header.js]: that was test, removed!

File: app/directives/meetings/conference-header.html [hunk 2]
```diff
@@ -186,123 +186,227 @@
...
+                </div>
+                
+                <div ng-if="meeting.conference.menus" style="width:100%">
+                    <table class="meetingnav-table">
+                        <tr>
+                           
+                            <td class="meetingnav-buttongroup menu {{::(cssClass(mainMenu.code) + ' ' + mainMenu.style)}}" ng-repeat="mainMenu in meeting.conference.menus" 
+                                ng-class="{'single-menu' : (!(mainMenu.menus||[]).length || (mainMenu.isExpandable && !mainMenu.isExpanded)), 
+                                    'no-left-border':$index==0, 'hidden': mainMenu.isHidden,
+                                    selected:meetingNavCtrl.isSelected('{{::mainMenu.url}}', {{::mainMenu.exactSelection}}) }"
```
Comment (sbilodeau) [inline app/directives/meetings/conference-header.html]: I think it's wrong to have `{{ }}` in function fo the ng-class

File: app/directives/meetings/conference-header.html [hunk 3]
```diff
@@ -186,123 +186,227 @@
...
+                        </tr>
+                    </table>
+               </div>
            </div>
        </div>
    </div>
+   <div id="menu-templates">
+        <script type="text/ng-template" id="menu">
+            <div class="menu-title">                    
+                <a  data-toggle="{{::(menu.links ? 'dropdown' : '')}}" ng-if="::menu.xsUrl||menu.url" ng-href="{{::menu.xsUrl||menu.url}}#{{meetingNavCtrl.hash()}}" class="link" target="{{::menu.target}}"></a>
```
Comment (sbilodeau) [inline app/directives/meetings/conference-header.html]: what us xsUrl? do we have different lick for different screen size?

Comment (blaisefonseca) [reply app/directives/meetings/conference-header.html]: its my way around, when on small screen the main menu becomes clickable and the user is redirected to first sub-menu

Comment (blaisefonseca) [reply app/directives/meetings/conference-header.html]: this actually makes the main menu clickable on big screen too even when the url is not mentioned, but i think its ok, no?

Comment (sbilodeau) [reply app/directives/meetings/conference-header.html]: ok. as long as it's not something stored in the DB

**Pull request merged**